blocks/button-group/button-group.js

```diff
@@ -0,0 +1,21 @@
+export default async function decorate(block) {
+  const buttons = [...block.children].map((row) => ({
+    anchorNode: row.children[0].firstChild.firstChild,
+    buttonStyle: row.children[1].firstChild,
+    altText: row.children[2].firstChild
+  }));
```

Comment:
I really like this map solve for making the components and I've stolen it for my own work.

Reply:
🙇